src/Background/estimators.jl

```diff
@@ -133,3 +133,61 @@ end
 
 MMM() = MMM(3, 2)
 estimate_background(alg::MMM, data; dims = :) = alg.median_factor * median(data, dims = dims) - alg.mean_factor * mean(data, dims = dims)
+
+"""
+    Biweight_location(M=nothing, c=6.0)
+
+Estimate the background using robust statistic biweight location.
+
+# Example
+```jldoctest
+julia> x = ones(3,5);
+
+julia> estimate_background(Biweight_location, x)
+1.0
+
+julia> estimate_background(Biweight_location, x, dims = 1)
+1×5 Array{Float64,2}:
+ 1.0  1.0  1.0  1.0  1.0
+ ```
+"""
+struct Biweight_location{T <: Number} <: BackgroundEstimator
+    M::Union{Nothing, T}
+    c::T
+end
+
+Biweight_location() = Biweight_location(nothing, 6.0)
+
+function Biweight_util(data::Array{T}, M::Union{Nothing, T}, c::T) where T <: Number
+
+    if M == nothing
+        M = median(data)
+    end
```

Comment:
Typically, for testing if something is nothing we use `===` (or in julia >1.1 `isnothing`)
```julia
M = M === nothing ? median(data) : M
```